www/README.md

```diff
@@ -1,74 +1,442 @@
+
+### Update the InSpec Repository Module In `chef/chef-web-docs`
+
+We use [Hugo modules](https://gohugo.io/hugo-modules/) and [vendoring](https://gohugo.io/hugo-modules/use-modules/#vendor-your-modules)
+to build Chef's documentation from multiple repositories. The Hugo modules are
+pinned to commits in each repository, and the documentation files from those
+repositories and commits are vendored in `chef/chef-web-docs`. When the documentation
+is updated in a repository, those changes won't appear in docs.chef.io until the
+Hugo modules are updated to the new commit and then vendored files are updated.
+
+This can be useful if you want to update documentation in `inspec/inspec`, but
+don't want it to appear on docs.chef.io until a later date.
+
+To update the Hugo module for documentation in `inspec/inspec`:
+
+1. Make sure your documentation changes are merged into master in `inspec/inspec`.
+1. Contact your friendly local Docs Team who will update the InSpec Hugo module for you.
+
+Or, for the adventurous:
+
+1. Make sure your documentation changes are merged into master in `inspec/inspec`.
+1. On a local clone of `chef/chef-web-docs` run:
+   1. `hugo mod get github.com/inspec/inspec/www`
+   1. `hugo mod tidy`
+   1. `hugo mod vendor`
+1. Submit a pull request to `chef/chef-web-docs`.
+
+This will updated the InSpec vendored files in `chef-web-docs/_vendor/github.com/inspec/inspec/www`,
+and update the commits in the go.mod and go.sum files in chef-web-docs.
+
+## Local Development Environment
+
+We use [Hugo](https://gohugo.io/), [Go](https://golang.org/), and[NPM](https://www.npmjs.com/)
+to build the Chef Documentation website. You will need Hugo 0.61 or higher
+installed and running to build and view our documentation properly.
+
+To install Hugo, NPM, and Go on Windows and macOS:
+
+- On macOS run: `brew install hugo node go`
+- On Windows run: `choco install hugo nodejs golang -y`
+
+To install Hugo on Linux, run:
+
+- `apt install -y build-essential`
+- `snap install node --classic --channel=12`
+- `snap install hugo --channel=extended`
+
+## Hugo Theme
+
+We use a git submodule to grab the Hugo theme from the `chef/chef-web-docs` repository.
+
+## Preview Workstation Documentation
+
+There are three ways to preview the documentation in `inspec`:
+
+- submit a PR
+- `make serve`
+- preview content from `chef/chef-web-docs`
+
+### Submit a PR
+
+When you submit a PR to `inspec/inspec`, Netlify will build the documentation
+and add a notification to the GitHub pull request page. You can review your
+documentation changes as they would appear on docs.chef.io.
+
+### make serve
+
+`make serve` will preview the documentation that only exists in `inspec/inspec/www`.
+This also shows a preview page that includes page metadata which can be useful
+for changing where a page exists in the left navigation menu.
+
+To build the docs and preview locally:
+
+- Run `make serve`
+- go to http://localhost:1313
```

Comment:
This URL leads to a page with 

> Below is a list of all of the pages in the Inspec documentation. It lists the page name, the file name, if the page exists in the inspec menu, and metadata about the page's menu entry if it is in the menu.
> 
> Pages that are not in the content/inspec subdirectory won't be copied to chef-web-docs. Those file names are highlighted in red.

I think this page should be ignored (it will be unreachable on chef.io) and the preview URL is `http://localhost:1313/inspec` ?

Reply:
Yep, that preview page only appears when you build the InSpec docs from the inspec repo but won't appear when the docs are built from docs.chef.io. I've added that in so the docs content in the repo can be checked quickly. I've got similar setups in workstation and desktop.